docs/usage/notifiers.md

```diff
@@ -0,0 +1,83 @@
+# Notifiers
+## Sample notifiers
```

Comment:
Not sure if it's necessary to rewrite the title and a sub title
![image](https://github.com/OpenCTI-Platform/docs/assets/53513584/5aa83141-8e82-4b57-aa40-d3521f4724fd)

Reply:
Example here
![image](https://github.com/OpenCTI-Platform/docs/assets/53513584/a59f6f4b-b575-47fc-b83e-3ef8dab7364b)